cluster/gce/gci/configure-helper.sh

```diff
@@ -1701,17 +1701,43 @@ function prepare-etcd-manifest {
   if [[ -n "${INITIAL_ETCD_CLUSTER_STATE:-}" ]]; then
     cluster_state="${INITIAL_ETCD_CLUSTER_STATE}"
   fi
+
+  # Configure mTLS for etcd peers.
   if [[ -n "${ETCD_CA_CERT:-}" && -n "${ETCD_PEER_KEY:-}" && -n "${ETCD_PEER_CERT:-}" ]]; then
     etcd_creds=" --peer-trusted-ca-file /etc/srv/kubernetes/etcd-ca.crt --peer-cert-file /etc/srv/kubernetes/etcd-peer.crt --peer-key-file /etc/srv/kubernetes/etcd-peer.key -peer-client-cert-auth "
     etcd_protocol="https"
   fi
 
-  # mTLS should only be enabled for etcd server but not etcd-events. if $1 suffix is empty, it's etcd server.
+  # In a zonal cluster, etcd_host_ip and host_ip are both the same as host VM's internal IP.
+  # In a regional cluster, a secondary alias IP range (usually 10.127.X.X) is allocated for etcd servers.
+  local host_ip=""
+  local etcd_addtional_listen_client_url=""
+  local etcd_addtional_listen_metrics_url=""
+  if [[ ${ETCD_LISTEN_ON_HOST_IP:-} == "true" ]] ; then
```

Comment:
Seems like this is just setting host_ip for the later blocks with the same switch. If combine them it makes the code simpler and ensures that the host_ip variable is not around longer than it should be.

Reply:
Also not strictly necessary but can we add this defaulting to false to https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-default.sh ?
Helps make the OSS behavior clear, documented and obvious where it twiddle it.

Reply:
I eliminated an if block. But the other one is very hard to eliminate without introducing more nested ifs.
Added default envvar value.